theano/gpuarray/extra_ops.py

```diff
@@ -22,8 +22,9 @@ class GpuCumsum(GpuKernelBase, Op):
     SUPPORTED_NDIMS = 3
     __props__ = ('axis',)
 
-    def __init__(self, axis):
```

Comment:
Yeah, this diff doesn't exist anymore. It has been removed. Earlier, i thought it was better to pass context_name